### ISSUE TITLE
Issue loading glb-file (binary gltf) since pixijs v6.4.2 (was working fine before)

### ISSUE THREAD

**Issue by bencresty** [pixijs/pixijs#8448]
### Expected Behavior

Working with both `pixijs` and `pixi3d` I'm importing 3d models from `glb` files. This was working fine in pixijs 6.0.0 to 6.3.2, but is broken since 6.4.2. I don't see any change to the loader in the changelog of 6.0.0 or 6.4.2, so this seems like a bug to me.

The loader still loads data from the `glb` file, but the returning object is pretty different.

Before the resource had a `gltf` property including the gltf-data. That needed `gltf` property with the model/scene inside doesn't exist in the returned resource anymore since pixi 6.4.2. The project didn't change, so the glb didn't change, neither did pixi3d. Only the pixi lib changed in the project.

### Current Behavior

Loader loads something from the glb, but the returned resource is missing the gltf property and datatype is now Text instead of ArrayBuffer.

### Possible Solution

Make the loader work as before :)

### Steps to Reproduce

* Install pixijs 6.3.2
* Load glb with the pixi loader
* See that it works (returns with a resource having the gltf property holding the data) --> see screenshot: data is of type arrayBuffer and the `gltf` property is a `glTFAsset`
![image](https://user-images.githubusercontent.com/69322733/175965892-3f47f976-eff8-4b03-927e-d2adf5110992.png)

* Upgrade pixi to 6.4.2
* Load glb with the pixi loader
* See that it stopped working, there's no gltf property in the result anymore --> see screenshot below: data isn't returned as ArrayBuffer anymore, but as text
![image](https://user-images.githubusercontent.com/69322733/175966345-64e5c613-57cb-4da6-acdf-010d3c07f9e9.png)

This also makes `pixi3d` stop working when trying to use the gltf as there's no `gltf` data in the resource. It needs that `glTFAsset` object:
![image](https://user-images.githubusercontent.com/69322733/175966760-2d531f81-9f2e-4c92-9117-96a8309370d9.png)

### Environment

- **`pixi.js` version**:  6.4.2
- **Browser & Version**:  chromium based (ms edge/chrome) latest
- **OS & Version**:  Windows

### glb to test with
[test-glb.zip](https://github.com/pixijs/pixijs/files/8992947/test-glb.zip)

**Comment by miltoncandelero** [comment]
this looks more like an issue with a gltf loader from pixi3d 🤔

**Comment by bigtimebuddy** [comment]
PixiJS has had issues upgrading (just bumping the version in the package an updating), have you tried a _clean_ install with 6.4.2? Does the issue still reproduce? 

If that doesn't do anything, could you create a repo that reproduces the problem so we can dig in more closely?

**Comment by bencresty** [comment]
> PixiJS has had issues upgrading (just bumping the version in the package an updating), have you tried a _clean_ install with 6.4.2? Does the issue still reproduce?

I've read about it somewhere, so yesterday I did test that. First I uninstalled pixi.js completely and after that I re-installed it. Same results. For the record: everytime I install 6.3.2 it just works fine again and that never needs an uninstallation. But when I go back to 6.4.2 it's broken again.

> If that doesn't do anything, could you create a repo that reproduces the problem so we can dig in more closely?

Right now it's happening in a full project full of configurations and build system and all and can't share that. Don't have the time to create a smaller project now, but I'll see what I can do.
